TestCases/parallel_regression.py

```diff
@@ -987,7 +987,7 @@ def main():
     channel_2D.cfg_dir   = "sliding_interface/channel_2D"
     channel_2D.cfg_file  = "channel_2D_WA.cfg"
     channel_2D.test_iter = 2
-    channel_2D.test_vals = [2.000000, 0.000000, 0.398053, 0.352779, 0.405462]
```

Comment:
I might have missed it in the diff but why did this Testcase change? It is only a minor change but I am curious where it was introduced

Reply:
The sliding mesh cases are sensitive to very minor things